install-gh-data.js

```diff
@@ -0,0 +1,69 @@
+// Copyright (C) 2015-2016 IBM Corporation and Others. All Rights Reserved.
```

Comment:
Not sure if this is right, or just got copied across from some of the other scripts

Reply:
it's copied from install-spawn